src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -28,7 +28,7 @@ object ConsumptionPlan : Plan(
             "",
             mapOf(
                 "data" to HandleConnection(
-                    StorageKeyParser.parse("reference-mode://{ramdisk://Thing}{ramdisk:///handle/my-handle-id}"),
+                    StorageKeyParser.parse("reference-mode://{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/Thing}{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/!:writingArcId/handle/my-handle-id}"),
```

Comment:
Niice! This looks good 👍

Reply:
Out of curiosity, why isn't there a storage key for the first Plan object? In a later PR, should we create a `Create` storage key object?

Reply:
are you expecting the capabilities-resolver to generate the `creatable` storage keys? i don't think it should be the one doing it, because this is a workaround specific to PA. probably we need to add some logic to storate-key-recipe-resolver to do this.
let's do it in follow up pr.

Reply:
I'm happy with that decision.